src/importer/tflite/ops/pad.cpp

```diff
@@ -62,3 +62,34 @@ DEFINE_TFLITE_LOWER(PADV2)
     link_input_tensor(&node->input(), op.inputs()->Get(0));
     link_output_tensor(op.outputs()->Get(0), &node->output());
 }
+
+DEFINE_TFLITE_LOWER(MIRROR_PAD)
+{
+    auto &input = get_tensor(op.inputs(), 0);
+    auto paddings = load_tensor<int32_t, 2>(get_tensor(op.inputs(), 1));
+    auto &options = *op.builtin_options_as_MirrorPadOptions();
+    auto tmp = options.mode();
+    pad_mode_t mode ;
+    switch(tmp)
+    {
+        case 0:
+            mode = pad_reflect;
+            break;
+        case 1:
+            mode = pad_symmetric;
+            break;
+        default:
+            throw("unsupport pad mode");
```

Comment:
throw a runtime_error instead of a raw string.

Reply:
Added